src/scss/layout/_main.scss

```diff
@@ -18,6 +19,10 @@
     grid-column: var(--column, page);
   }
 
+  > a:any-link {
+    justify-self: start;
+  }
+
```

Comment:
did we want all links to left align? Like the ones on the homepage in the sections with the ribbon headline are all now on the far left instead of at the bottom right of each section.

Reply:
oops! Guess I need to do something smarter here.